slp2mp4/slp-to-mp4.py

```diff
@@ -145,7 +161,13 @@ def record_folder_slp(slp_folder, conf):
         slp_file = os.path.join(in_file[0], in_file[1])
         out_file = os.path.join(OUT_DIR, out_files[index][0], out_files[index][1])
         if not os.path.exists(out_file):
-            record_file_slp(slp_file, out_file)
+            args.append((slp_file, out_file))
+
+    num_processes = get_num_processes(conf)
+
+    pool = multiprocessing.Pool(processes=num_processes)
+    pool.starmap(record_file_slp, args)
```

Comment:
starmap takes a function as the first argument and a list of tuples for the second.